sdk/daml-assistant/get-daml.sh

```diff
@@ -127,7 +127,7 @@ elif [ "$major" = "2" ] && [ "$minor" -ge "9" ]; then
       platform=
       ;;
     linux-aarch64)
-      platform=-aarch64
```

Comment:
This code path now always has the `platform` as empty. We don't support arch64 linux in 2.9 at all right now, so we should just eliminate this case.
I'd remove the case for `elif [ "$major" = "2" ] && [ "$minor" -ge "9" ]; then`, and change the previous `elif [ "$major" = "2" ] && [ "$minor" -le "8" ]; then` to simply `elif [ "$major" = "2" ]; then`

Reply:
@samuel-williams-da We're still running the `Linux (arm)` job on CI for `release/2.9.x`
e.g.
https://dev.azure.com/digitalasset/daml/_build/results?buildId=175828&view=logs&jobId=61ac79bb-734c-5c5d-b939-064aaa72fe3b

Reply:
Ah right sorry, we don't release any aarch64 artifacts in 2.9 is what I mean.